documentation/meta/ci_cd/jobs/preparation.md

```diff
@@ -38,3 +38,18 @@ of the last commit.
   image_tag: string // the tag to use for the Docker images
 }
 ```
+
+## `lint`
+
+Runs the linting steps defined in the repository's
+[pre-commit configuration](https://github.com/WordPress/openverse/blob/main/.pre-commit-config.yaml).
+This is executed via `just lint`.
+
+## `add-stack-label`
+
+_Note: Only runs on PRs._
```

Comment:
Because this is Sphinx, why not make the most of it?
````suggestion
```{note}
This job only runs on PRs.
```
````

Reply:
Ah, that's more appropriate 😄